src/server/serverless/api.ts

```diff
@@ -1,9 +1,13 @@
 import serverless, { Handler } from 'serverless-http'
 
 import bootstrap from '../bootstrap'
-const apiHandler = bootstrap().then((app) => serverless(app))
 
 export const handler: Handler = async (event, context) => {
-  const h = await apiHandler
-  return h(event, context)
+  const { app, sequelize } = await bootstrap()
+  const h = serverless(app)
+  try {
+    return await h(event, context)
+  } finally {
+    await sequelize.close()
```

Comment:
Should we add a `logger.info('Closed sequelize connection')`

Reply:
we could.. trying to avoid bringing it in as a module to depend on though